source/index.html.md.erb

```diff
@@ -31,6 +31,53 @@ To allow users to be able to rapidly test the new EMCS API, we have stubbed the
 
 You can simulate common error responses by sending messages with standard inputs. These are detailed in the [test scenarios](documentation/test-scenarios.html).
 
+## Push pull notification testing
+When testing your software, you can be sent automatic notifications using the Push Pull Notifications API.
+
+This is where notifications from EMCS are sent to your software in near real time.
+
+Only the consignor will receive Push Pull Notifications from the service.
+
+To get these automatic messages, you must:
+
+- subscribe to the [Push Pull Notifications Service](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/push-pull-notifications-api/1.0)
+- subscribe to the Excise Movement Control System (EMCS) API
+- configure the notification URL in the subscription configuration section of Developer Hub
+
+Already subscribed to the EMCS API?
+
+You will need to set up a Push Pull Notification API BoxID to receive the notifications.
+To do this go to the Manage API subscriptions page for your application in the Developer Hub.
+Go to the section Excise Movement Control System, and simply toggle from Yes to No then back to Yes.
+
+**Note:**
+if your notification web address is hosted by Amazon Web Services (AWS) then you must either use [edge-optimised custom domain names](https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-edge-optimized-custom-domain-name.html)
+or [regional custom domain names](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-regional-api-custom-domain-create.html).
+
+
+### Examples of Push Notification
+
+The following is an example of the push notification showing the link to the messageUri
+
+```
+{
+    "notificationId": "c7f5b4bb-15b8-4805-bde0-fec4e3499af5",
```

Comment:
I think the third party will receive only the message object and no the rest (notificationId, messageContentType, etc)

Reply:
sorry ignore my message above